src/crypto/PSAOperationalKeystore.cpp

```diff
@@ -209,5 +254,53 @@ void PSAOperationalKeystore::ReleasePendingKeypair()
     mIsPendingKeypairActive = false;
 }
 
+CHIP_ERROR PSAOperationalKeystore::MoveOpKeysFromKVSToITS(FabricIndex fabricIndex) const
```

Comment:
This should not mention "KVS", but rather "PersistentStorage" (which may not be backed by Kvs directly).